SPECS/libreswan/libreswan.spec

```diff
@@ -11,6 +11,7 @@
     INITSYSTEM=systemd \\\
     PYTHON_BINARY=%{__python3} \\\
     SHELL_BINARY=%{_bindir}/sh \\\
+    DEFAULT_DNSSEC_ROOTKEY_FILE="/var/lib/unbound/root.key" \\\
```

Comment:
In current version 4.7, this is the default value path[[link](https://github.com/libreswan/libreswan/blob/19eabcd8e2b052747617240b16ea683a912d45e7/mk/config.mk#L592)] but later libreswan's maintainers decided to not provide a default value[[link](https://github.com/libreswan/libreswan/issues/755) ] and explicitly specify the default per-os